windows/src/test/Makefile

```diff
@@ -4,7 +4,11 @@
 
 # ----------------------------------------------------------------------
 
+!ifdef NODELPHI
+TARGETS=.virtual
+!else
 TARGETS=test_i3633
+!endif
 CLEAN=test_i3633
```

Comment:
Does CLEAN need to go in the `!else` branch?

Reply:
CLEAN doesn't have any Delphi-specific build steps, so it should be okay.